spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -765,6 +776,7 @@ Cloud Datastore only supports filter components joined by AND, and the following
 * `is null`
 
 After writing a custom repository interface specifying just the signatures of these methods, implementations are generated for you and can be used with an auto-wired instance of the repository.
+Because of Cloud Datastore's requirement that explicitly selected fields must all appear in a composite index together, `find` name-based query methods are run as `SELECT *`.
```

Comment:
```suggestion
Cloud Datastore requires that explicitly selected fields must all appear in a composite index together. To avoid this restriction, name-based query `find` methods are run as `SELECT *`.
```

Reply:
Hmmm I'm not sure this is better.